### PR TITLE
feat(ui5-product-switch): Selected state added to an item

### PR DESCRIPTION
**Pull request by PetyaMarkovaBogdanova**
Now ui5-product-switch-item recieves a selected state on click or keydown, which is unique for an ui5-product-switch collection

### CODE REVIEW

File: packages/fiori/src/ProductSwitch.js [hunk 1]
```diff
@@ -156,6 +156,13 @@ class ProductSwitch extends UI5Element {
 		}
 	}
 
+	handleProductSwitchItemClick(event) {
+		event.target.parentNode.querySelectorAll("ui5-product-switch-item").forEach(item => {
```
Comment (ilhan007) [inline packages/fiori/src/ProductSwitch.js:160]: To get the ProductSwitch items, you just need to use "this.items" and this will become 
this.items.forEach(item => { item.selected = false;});

Comment (ilhan007) [reply packages/fiori/src/ProductSwitch.js:160]: Should not this behaviour also happen on SPACE/ENTER?

File: packages/main/test/pages/AvatarGroup.html [hunk 2]
```diff
@@ -26,6 +26,7 @@
 		#individual-pop .avatar-slot,
 		#groupPop .avatar-slot {
 			display: inline-block;
+			vertical-align: middle;
```
Comment (nnaydenow) [inline packages/main/test/pages/AvatarGroup.html:29]: seems to be not related to this change

**Comment by nnaydenow** [comment]
Shift + Space should not change the selection. Selected item should have --sapList_Hover_SelectionBackground background color when it is hovered.

**Review by ilhan007 — APPROVED** [review]

**Review by ilhan007 — APPROVED** [review]
